### PR TITLE
13-auto: Fix typos, formatting

### PR DESCRIPTION
**Pull request by ddvlad**
Signed-off-by: Vlad Dogaru <ddvlad@gmail.com>

### CODE REVIEW

**Comment by ddvlad** [comment]
This is literally my first time using github for a pull request, please be gentle.

I also have a few other suggestions that can't be implemented in a simple chapter review, how should I raise them?   Issue tracker here? https://github.com/systems-cs-pub-ro/uso-book/issues

**Comment by ddvlad** [comment]
Sper că e clar, de asemenea, că toate schimbările sunt simple sugestii, și că nu am uitat cum se vorbește românește :D

File: book/chapters/13-auto/contents.tex [hunk 1]
```diff
@@ -398,7 +396,7 @@ \subsection{Variabile}
 Adesea, vom folosi variabile pentru menține o informație comună mai multor comenzi, similar unui limbaj de programare.
 De exemplu, variabila \texttt{i} din scriptul \file{wiki-cat} din \labelindexref{Listing}{lst:auto:wiki-cat} este folosită împreună cu \cmd{for}, cu \cmd{if} / \cmd{test}, cu \cmd{cat} și cu \cmd{basename}.
 
-Valoarea unei variabile poate fi inițializată și folosită prin operație simplă de atribuire, respectiv prefizarea ei cu simbolul \texttt{\$} (\textit{dollar}).
+Valoarea unei variabile poate fi inițializată și folosită prin operația simplă de atribuire, respectiv prin prefizarea ei cu simbolul \texttt{\$} (\textit{dolar}).
```
Comment (razvand) [inline book/chapters/13-auto/contents.tex:399]:
```suggestion
Valoarea unei variabile poate fi inițializată și folosită prin operația simplă de atribuire, respectiv prin prefixarea ei cu simbolul \texttt{\$} (\textit{dolar}).
```

File: book/chapters/13-auto/contents.tex [hunk 2]
```diff
@@ -445,7 +443,7 @@ \subsection{Variabile}
 Construcțiile aferente parametrilor unui script shell sunt prezentate în \labelindexref{Tabelul}{tab:auto:script-params}.
 Construcțiile \texttt{\$*} și \texttt{\$@} au, în aparență același efect.
 Diferență constă în escaparea celor două construcții prin ghilimele.
-Astfel construcția \texttt{"\$*"} se va expanda la o singură valoare care conține spații între parametrii scriptului; în vreme ce construcția \texttt{"\$@"} se va expanda la o listă de valori escapate fiecare prin ghilimele.
+Astfel construcția \texttt{\$*} se va expanda la o singură valoare care conține spații între parametrii scriptului; în vreme ce construcția \texttt{\$@} se va expanda la o listă de valori escapate fiecare prin ghilimele.
```
Comment (razvand) [inline book/chapters/13-auto/contents.tex:446]:
```suggestion
Astfel, construcția \texttt{\$*} se va expanda la o singură valoare care conține spații între parametrii scriptului, în vreme ce construcția \texttt{\$@} se va expanda la o listă de valori escapate fiecare prin ghilimele.
```

File: book/chapters/13-auto/contents.tex [hunk 3]
```diff
@@ -583,7 +582,7 @@ \subsubsection{if}
         \cmd{test -z "\$store"} &
         verifică dacă variabila \texttt{store} nu conține nici o informație (este un șir vid) \\
 
-        \cmd{test ! -z "\$store"} &
+        \cmd{test !\@ -z "\$store"} &  % \@ este necesar după !, altfel LaTeX consideră că se termină o frază și bugetează spațiu în plus.
```
Comment (razvand) [inline book/chapters/13-auto/contents.tex:585]: TIL

File: book/chapters/13-auto/contents.tex [hunk 4]
```diff
@@ -1217,8 +1215,9 @@ \subsection{Lizibilitate}
 set completion-ignore-case On
 END
 
-bc <<< "2^10"
+bc <<< "2^\verb+^+^10"
 \end{screen}
+% I double dog dare you to come up with ^\verb+^+^ for making a ^ inside the screen environment without looking at the .sty.
```
Comment (razvand) [inline book/chapters/13-auto/contents.tex:1220]: Te uiți în celelalte capitole, că am mai făcut asta undeva :1st_place_medal:

**Review by razvand — APPROVED** [review]
Awesome review. Mersi. Am lăsat câteva sugestii. După ce le integrezi, facem merge.

**Comment by razvand** [comment]
> This is literally my first time using github for a pull request, please be gentle.
> 
> I also have a few other suggestions that can't be implemented in a simple chapter review, how should I raise them? Issue tracker here? https://github.com/systems-cs-pub-ro/uso-book/issues

@ddvlad , yes, please raise issues.

**Pull request merged**